packages/e2e-tests/src/tests/SessionSigs.spec.ts

```diff
@@ -0,0 +1,825 @@
+
+  devEnv.releasePrivateKeyFromUser(alice);
+
+  expect(res?.response).toBeDefined();
+  expect(res?.logs).toBeDefined();
+  expect(res?.logs.includes('hello world')).toBeTruthy();
+  expect(res?.success).toBeTruthy();
+};
+
+const executeJsSigning = async (
+  devEnv: TinnyEnvironment,
+  generator: (
+    devEnv: TinnyEnvironment,
+    person: TinnyPerson,
+    resources?: LitResourceAbilityRequest[]
+  ) => Promise<SessionSigsMap | undefined>
+): Promise<void> => {
+  const alice = await devEnv.createRandomPerson();
+  const litActionSessionSigs = await generator(devEnv, alice, [
+    {
+      resource: new LitPKPResource('*'),
+      ability: LitAbility.PKPSigning,
+    },
+    {
+      resource: new LitActionResource('*'),
+      ability: LitAbility.LitActionExecution,
+    },
+  ]);
+
+  const res = await devEnv.litNodeClient?.executeJs({
+    sessionSigs: litActionSessionSigs,
+    code: `(async () => {
+      const sigShare = await LitActions.signEcdsa({
+        toSign: dataToSign,
+        publicKey,
+        sigName: "sig",
+      });
+    })();`,
+    jsParams: {
+      dataToSign: alice.loveLetter,
+      publicKey: alice.authMethodOwnedPkp?.publicKey,
+    },
+  });
+
+  devEnv.releasePrivateKeyFromUser(alice);
+
+  expect(res?.signatures?.sig.r).toBeDefined();
+  expect(res?.signatures?.sig.s).toBeDefined();
+  expect(res?.signatures?.sig.dataSigned).toBeDefined();
+  expect(res?.signatures?.sig.publicKey).toBeDefined();
+};
+
+const executeJsSigningParallel = async (
+  devEnv: TinnyEnvironment,
+  generator: (
+    devEnv: TinnyEnvironment,
+    person: TinnyPerson,
+    resources?: LitResourceAbilityRequest[]
+  ) => Promise<SessionSigsMap | undefined>
+): Promise<void> => {
+  devEnv.setUnavailable(LIT_TESTNET.MANZANO);
+
+  const alice = await devEnv.createRandomPerson();
+
+  const litActionSessionSigs = await generator(devEnv, alice);
+
+  const fn = async (index: number) => {
+    return await devEnv.litNodeClient?.executeJs({
+      sessionSigs: litActionSessionSigs,
+      code: `(async () => {
+      const sigShare = await LitActions.signEcdsa({
+        toSign: dataToSign,
+        publicKey,
+        sigName: "sig",
+      });
+    })();`,
+      jsParams: {
+        dataToSign: alice.loveLetter,
+        publicKey: alice.authMethodOwnedPkp?.publicKey,
+      },
+    });
+  };
+
+  devEnv.releasePrivateKeyFromUser(alice);
+
+  const res = await Promise.all([fn(1), fn(2), fn(3)]);
+
+  res.forEach((r) => {
+    expect(r?.signatures?.sig.r).toBeDefined();
+    expect(r?.signatures?.sig.s).toBeDefined();
+    expect(r?.signatures?.sig.dataSigned).toBeDefined();
+    expect(r?.signatures?.sig.publicKey).toBeDefined();
+  });
+};
+
+const decryptString = async (
+  devEnv: TinnyEnvironment,
+  generator: (
+    devEnv: TinnyEnvironment,
+    person: TinnyPerson,
+    resources?: LitResourceAbilityRequest[]
+  ) => Promise<SessionSigsMap | undefined>
+): Promise<void> => {
+  devEnv.setUnavailable(LIT_TESTNET.MANZANO);
```

Comment:
🔪 ideally this would be replaced with wrapping the test definition function call itself in a conditional check, see above comment.

That being said, do we _really_ never run any test that uses this function on manzano -- and should it be disabled on `datil-test` if it was disabled on `manzano`?